src/models/tools/unknown-content.ts

```diff
@@ -9,5 +10,8 @@ registerToolContentInfo({
   id: kUnknownToolID,
   tool: "unknown",
   modelClass: UnknownContentModel,
-  defaultContent
+  defaultContent,
+  // TODO: should really have a separate unknown tool that shows an "unknown tile" message
+  Component: PlaceholderToolComponent,
```

Comment:
Exactly, that was my intent all along with the `UnknownContentModel`, that it could be instantiated with an `UnknownComponent` that would put up some sort of `Unknown Tile` or `Tile Unavailable` message but preserve the content so that if the same document were later opened up in a situation in which the missing tool was now available, everything would work seamlessly. Actually implementing the `UnknownComponent` has never been a high priority, however, because all of the tile types have been built-in so there's never been a scenario in which the `UnknownComponent` would be applicable. I think it's worth adding a PT story for the `UnknownComponent`.

Reply:
Done: https://www.pivotaltracker.com/story/show/180185884

I copied this comment from where this was handled before in the mapping in ToolTileComponent.